index.js

```diff
@@ -60,11 +61,15 @@ function setEventNames(sqsConfig) {
   );
 }
 
-async function sendToSqs({ record, params }) {
+async function sendToSqs({ record, params, logPayloadTransformer = _.identity }) {
   const message = params.bodyHandler(record);
   const MessageBody = JSON.stringify(message);
 
-  params.logger.info('DynamoDB Record: %j', record);
+  try {
+    params.logger.info('DynamoDB Record: %j', logPayloadTransformer(record));
+  } catch (err) {
+    params.logger.error({ err }, 'Error logging DynamoDB record');
```

Comment:
how does this error show in logs? I'm not sure how it will display `{ err }`. We use `%j` in the other log message, I wonder if we should use it here too?

Reply:
Here's a sample of how it looks (copied the style from elsewhere in the file).

![image](https://github.com/user-attachments/assets/ec66b494-f388-477f-b137-85cc3b1d614a)

but maybe we should log the record too? Then we could debug the transformer